### PR TITLE
Add flink container

### PR DESCRIPTION
**Pull request by hongzhanchen**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ env:
   - DOCKERFILE_DIR=redis
   - DOCKERFILE_DIR=ruby
   - DOCKERFILE_DIR=tensorflow
+  - DOCKERFILE_DIR=flink
```
Comment (gtkramer) [inline .travis.yml]: Please include flink in sorted order

File: flink/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,41 @@
+###############################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
```
Comment (gtkramer) [inline flink/Dockerfile]: @bryteise do we need this license notice here?

File: flink/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Start ruby Container
+-----------------------
+```
+docker run  -d clearlinux/flink
+```
+
+How to use this image
+---------------------
+- See ``How to use this image`` section of the official ruby image [page](https://hub.docker.com/_/flink).
```
Comment (gtkramer) [inline flink/README.md]: I don't think this is a ruby image ;)

File: flink/Dockerfile (hunk 2 above)
Comment (bryteise) [reply flink/Dockerfile]: What is the license for? The Dockerfile? That seems odd. I don't see why having a license there makes sense.

Comment (hongzhanchen) [reply flink/Dockerfile]: I referred to https://github.com/docker-flink/docker-flink/blob/f96f07a516dbd4d9544d22c5272006ccd0b988d6/1.7/hadoop24-scala_2.11-debian/Dockerfile which have this license. If we do not need it , I will modify it. Thanks for your comments.

**Pull request closed without merging**